Orange/widgets/data/owimpute.py

```diff
@@ -348,6 +375,20 @@ def imputed_vars_for_column(self, column):
         else:
             assert False
 
+    def _handle_value_errors(func):
+        @wraps(func)
+        def wrapper(self, *args, **kwargs):
+            global error_shown
+            try:
+                func(self, *args, **kwargs)
+                self.error()
+                error_shown = False
+            except ValueError as ve:
```

Comment:
When can this error happen? If it's only when a model-based imputer is chosen and it doesn't handle the (target) variables of that type (e.g. linear regression can't impute discrete values), this could be mentioned somewhere, I guess.